tools/autograd/derivatives.yaml

```diff
@@ -2053,6 +2056,7 @@
 - name: threshold_backward(Tensor grad_output, Tensor self, Scalar threshold) -> Tensor
   grad_output: threshold_backward(grad, self, threshold)
   self: zeros_like(grad)
+  result: zeros_like(self_t) + threshold_backward(grad_output_t, self_p, threshold)
```

Comment:
What is the zeros_like for here?

Reply:
the one in the native namespace?

Reply:
Ho I meant why are you adding 0s to something? Why not just have the something?

Reply:
Ah it should be for when the self has tangent but grad_output does not, then LHS is batched but RHS is not, so we'd like to preserve the batchedness.